index.js

```diff
@@ -209,7 +210,7 @@ function mountfs_path(devPath, path, type, flags, extras, callback)
   catch(error)
   {
     // catch everything, but not "Error no Entry"
-    if(err.code !== 'ENOENT') return callback(error)
+    if(error.code !== 'ENOENT') return callback(error)
 
     // mount the filesystem
     if(devPath)
```

Comment:
Are you talking about checking that the path actually exists? I think this is not needed, since `mount()` already check it.

Also the mount module needs a refactor since `dev` is not needed in some filesystems (the ones that don't use a real device or overlay on top another directory), and in fact the BSD version (and OSX) of the function consider it a filesystem option, not an argument of the function.